docs/sql-reference/sql-functions/array-functions/element_at.md

```diff
@@ -0,0 +1,31 @@
+# element_at
+
+## Description
+
+Returns the array element at the specific position. if any parameter is NULL, the result is also NULL.
+
+It is the alias of `[]` to get an element from an array.
```

Comment:
why not use the `[]`

Reply:
what's the use case of this `element_at` function

Reply:
trino.